### DISCUSSION TITLE
125W DC Power zeigt aber nur 60W Output Power

### DISCUSSION THREAD

**Discussion by Amutep** [tbnobody/OpenDTU#1413] [General]
Hallo Leute

Wenn ich aus meinem Solar Flow aktuell 120W bereitstelle zeigt mir die DTU folgende Werte

![Screenshot 2023-10-08 120056](https://github.com/tbnobody/OpenDTU/assets/145967865/5a57e2df-db27-46a7-a662-79178aac6c93)

Laut Anzeige 60Watt Gesamtleistung aber laut Shelly und Tapo Steckdose kommen die eingestellten 120Watt raus?
Auch wenn ich beide Module zusammenzähle also Links und Rechts kommt das richtige raus.

Sieht wie ein Anzeigefehler aus im unteren Bereich. 110W abwärts

lg

Eike

**Comment by Solarteur** [comment]
Das wurde schon öfters beschrieben, z. B. hier: 
https://github.com/tbnobody/OpenDTU/issues/1102#issuecomment-1621138846

In dem Link ist auch ausgeführt, dass bei einer Akku-Spannung von ca. 48V das "Problem" nicht mehr besteht.

**Reply by Amutep** [reply]
danke dir =)
